src/internal/build.ts

```diff
@@ -154,7 +154,7 @@ export class PlaygroundBuild {
 
   private _errorPendingFileRequests(error: HttpError) {
     for (const file of this._files.values()) {
-      if (!file.resolved) {
+      if (!file.settled) {
```

Comment:
No action required - just a question: If a file has already settled, does that mean it succeeded?

Reply:
In this case, yeah! Either the worker produces a file by this name and resolves with the file, or the worker completes the build but never produces one of the files that the iframe requested, in which case we resolve it with this 404 error object instead.